.github/workflows/common.sh

```diff
@@ -157,3 +157,43 @@ function apply_patches() {
   git am "${SDK_OUTER_SRCDIR}"/third_party/coreos-overlay/0*.patch
   rm -f "${SDK_OUTER_SRCDIR}"/third_party/coreos-overlay/0*.patch
 }
+
+# Return 1 if VER1 >= VER2
+function semver_is_bigger() {
+	local VER1="${1}"
+	local VER2="${2}"
+
+	if [[ "${VER1}" = "$(echo -e "${VER1}\n${VER2}" | sort -V | tail -n1)" ]]; then
+		return 1
+	fi
+
+	return 0
```

Comment:
```suggestion
		return 0
	fi

	return 1
```

Let's keep the semantics of 0 exit code meaning `true`, so that you can use `if semver_is_bigger; then` as in `if true; then`.

Reply:
Done